src/notification/Notification.js

```diff
@@ -0,0 +1,66 @@
+import GrouperClient from "../structures/GrouperClient";
+
+export default class Notification {
+    /**
+     * Creates a new store for commands
+     * 
+     * @param {GrouperClient} client 
+     */
+    constructor(client, guild, channel) {
+        /**
+         * GrouperClient
+         * 
+         * @type {GrouperClient}
+         */
+        this.client = client;
+        let supportGuild = null;
```

Comment:
Perhaps `let supportGuild = client.guilds.find(g => g.id == guild);` as opposed to the forEach. I'd test this snippet before you implement it though.

Reply:
It's a hashmap, the operation is O(1) if just access the key directly (`.get`) to get the guild.